plugins/kubernetes/test/models/kubernetes/template_filler_test.rb

```diff
@@ -187,9 +181,25 @@
       end
 
       it "fails when vault is not configured" do
-        Samson::Secrets::VaultClient.client.expects(:client).returns(nil)
-        e = assert_raises { template.to_hash }
-        e.message.must_equal "Could not find Vault config for pod1"
+        with_env('SECRET_STORAGE_BACKEND': "SecretStorage::HashicorpVault") do
+          Samson::Secrets::VaultClient.client.expects(:client).raises("Could not find Vault config for pod1")
+          e = assert_raises { template.to_hash }
+          e.message.must_equal "Could not find Vault config for pod1"
+        end
+      end
+
+      it "get the VAULT_ADDR in it's env" do
```

Comment:
when test description gives background information instead of repeating the test code it is easier to understand for newcomers and can reveal logical errors in the test code itself:
`it "adds then vault server address to the containers env"`